lib/trino-parquet/src/main/java/io/trino/parquet/reader/decoders/ValueDecoders.java

```diff
@@ -469,7 +469,7 @@ public void skip(int n)
         };
     }
 
-    public ValueDecoder<long[]> getInt64TimestampMillsToShortTimestampDecoder(ParquetEncoding encoding)
+    public ValueDecoder<long[]> getInt64TimestampMillsToShortTimestampDecoder(ParquetEncoding encoding, DateTimeZone timeZone)
```

Comment:
Pre-existing - separate commit 

Rename `getInt64TimestampMillsToShortTimestampDecoder` to `getInt64TimestampMillisToShortTimestampDecoder`